task/01-strings-tasks.js

```diff
@@ -238,8 +267,16 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+   // throw new Error('Not implemented');  
+   
+  if( typeof value === "string" || value instanceof String){
+  return true; 
+  }
+  else{ return false; }
```

Comment:
We can just ```( typeof value === "string" || value instanceof String){``` with the same result

Please, see https://javascript.info/logical-operators